### PR TITLE
Adding after_commit and after_rollback for class

### PR DESCRIPTION
**Pull request by zaratan**
No tests added since I have no time to rewrite them in minitest :/

### CODE REVIEW

File: test/after_commit_test.rb [hunk 1]
```diff
@@ -119,6 +136,8 @@ def teardown
     end
   end
 
+  def test_after
+
```
Comment (mtrepanier) [inline test/after_commit_test.rb]: !?

Comment (zaratan) [reply test/after_commit_test.rb]: woops. left over and will be fixed :D but since I can't run the tests at all…

**Comment by francoisjacques** [comment]
Test::Unit doesn't work with ruby 2.3?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```
Comment (francoisjacques) [inline Gemfile:3]: wip?

Comment (zaratan) [reply Gemfile:3]: Nope. This is a standard Gemfile for a Gem.

**Review by francoisjacques — COMMENTED** [review]
Merge when you want.

We should try to have this pushed upstream too.

**Comment by zaratan** [comment]
And no… Test::Unit have been removed from ruby since `2.2.0`… And even with `2.1.10` the tests doesn't seems to work on master…

**Comment by zaratan** [comment]
Will try to push upstream as soon as I have time to migrate to minitest

**Pull request merged**